tests/RPCServer.test.ts

```diff
@@ -1016,7 +1016,86 @@ describe(`${RPCServer.name}`, () => {
 
       // Longer timeout is set to server's default
       const ctxLong = await ctxLongProm.p;
-      expect(ctxLong.timer.delay).toEqual(50);
+      expect(ctxLong.timer.delay).toEqual(26);
+      waitProm.resolveP();
+      await rpcServer.stop({ force: true });
+    }
+  });
+  test('handler overrides timeout - greater value', async () => {
```

Comment:
Do you have the lesser test too? Try to make the test names consistent.

Reply:
I suggest breaking up the tests into separate tests to be clear. And I need tests for all 4 cases of calls.